crates/executor/benches/execution.rs

```diff
@@ -1,11 +1,11 @@
-#![allow(missing_docs)]
 //! Benches for the [StatelessL2BlockExecutor] implementation.
 
+#![allow(missing_docs)]
```

Comment:
```suggestion
```

Reply:
Keeping as these get deleted in the next upstack PR.